popup.js

```diff
@@ -1,38 +1,46 @@
 let active = document.getElementById('active');
+const reloadInput = document.getElementById('reload-int-input');
 
+// Maintain checked input box if this window has active reload
 chrome.tabs.query({ currentWindow: true, active: true }, function (tabs) {
   let tab = tabs[0];
   chrome.storage.local.get(tab.id.toString(), (items) => {
       if(items[tab.id]){
           console.log("found!");
           active.checked = true;
-          
       }
   });
 });
 
+// Save current window tab to Chrome storage
 active.onchange = function(event) {
-  if(event.target.checked){
-    let interval = 10000;
+  if (event.target.checked) {
     
     chrome.tabs.query({ currentWindow: true, active: true }, function (tabs) {
-      console.log(tabs[0]);
       let tab = tabs[0];
-      let tostore = {};
-      tostore[tab.id] = true;
-      chrome.storage.local.set(tostore);
+      let storedObj = {};
+      storedObj[tab.id] = true;
+      chrome.storage.local.set(storedObj);
     });
+
     chrome.browserAction.setBadgeText({text: 'ON'});
-    
-  }
-  else{
+
+  } else {
+
     chrome.tabs.query({ currentWindow: true, active: true }, function (tabs) {
-      console.log(tabs[0]);
       let tab = tabs[0];
-      let tostore = {};
-      tostore[tab.id] = false;
-      chrome.storage.local.set(tostore);
+      let storedObj = {};
+      storedObj[tab.id] = false;
+      chrome.storage.local.set(storedObj);
     });
+
     chrome.browserAction.setBadgeText({text: 'OFF'});
+
   }
+}
+
+// Save reload interval parameter to Chrome storage
+reloadInput.onchange = function(event) {
+  const numInput = parseInt(reloadInput.value);
+  chrome.storage.local.set({ input: numInput });
 }
```

Comment:
There was indeed no newline at the end, but would you mind adding one?

Reply:
I removed the entire event listener and instead of listening for changes in the number input, I chose to save the input when the checkbox is pressed.